crates/ruff_linter/src/rules/eradicate/detection.rs

```diff
@@ -1,43 +1,45 @@
 /// See: [eradicate.py](https://github.com/myint/eradicate/blob/98f199940979c94447a461d50d27862b118b282d/eradicate.py)
+use aho_corasick::AhoCorasick;
 use once_cell::sync::Lazy;
 use regex::Regex;
 
 use ruff_python_parser::parse_suite;
 
+static CODE_INDICATOR: Lazy<AhoCorasick> = Lazy::new(|| {
+    AhoCorasick::new([
+        "(", ")", "[", "]", "{", "}", ":", "=", "%", "print", "return", "break", "continue",
+        "import",
+    ])
+    .unwrap()
+});
```

Comment:
I actually don't know how much this improves performance over a manual iteration. Having trouble getting this function to show up in Instruments.

Reply:
You may need a test file with a ton of comments